configure.ac

```diff
@@ -106,15 +106,15 @@ fi
 AC_SUBST([enable_beta])
 AM_CONDITIONAL([ENABLE_BETA], [test "x$enable_beta" = "x1"])
 
-AC_ARG_ENABLE([procfs],
-  [AS_HELP_STRING([--disable-procfs], [Disable use of procfs for use on operating systems other than standard Linux])],
-[if test "x$enable_procfs" = "xno" ; then
-  enable_procfs="0"
+AC_ARG_ENABLE([systemd],
+  [AS_HELP_STRING([--disable-systemd], [Disable the use of the sd_bus interface of libsystemd to support the ServiceIOGroup])],
+[if test "x$enable_systemd" = "xno" ; then
+  enable_systemd="0"
 else
-  enable_procfs="1"
+  enable_systemd="1"
 fi
 ],
-[enable_procfs="1"]
+[enable_systemd="1"]
```

Comment:
If this is the default, this is going to break the CI pipeline until we update the requirements.  Fine for now.